### PR TITLE
EN-1626: set local statement_timeout based on queryTimeoutMs param, i…

### PR DESCRIPTION
**Pull request by meganschneider**
…f given

### CODE REVIEW

File: soql-server-pg/src/main/scala/com/socrata/pg/query/DataSqlizerQuerier.scala [hunk 1]
```diff
@@ -69,8 +71,23 @@ trait DataSqlizerQuerier[CT, CV] extends AbstractRepBasedDataSqlizer[CT, CV] wit
...
+      stmt.execute(s)
+    }
+  }
+
+  def setTimeout(timeoutMs: String) = s"SET LOCAL statement_timeout TO $timeoutMs"
+  def resetTimeout = "SET LOCAL statement_timeout TO DEFAULT"
+
+  private def executeSql(conn: Connection, pSql: ParametricSql, timeout: Option[Duration]): ResultSet = {
     try {
+      if (timeout.isDefined && timeout.get.isFinite()) {
```
Comment (aerust) [inline soql-server-pg/src/main/scala/com/socrata/pg/query/DataSqlizerQuerier.scala]: Why are you checking that is is finite here? (just curious)

**Comment by aerust** [comment]
lgtm :fish:

File: soql-server-pg/src/main/scala/com/socrata/pg/query/DataSqlizerQuerier.scala (hunk 1 above)
Comment (meganschneider) [inline soql-server-pg/src/main/scala/com/socrata/pg/query/DataSqlizerQuerier.scala]: is not infinite or NaN :) (also cause that seems to be our common check for "hey is this a real thing", sprinkled through code, but mostly the first bit)

Comment (meganschneider) [inline soql-server-pg/src/main/scala/com/socrata/pg/query/DataSqlizerQuerier.scala]: Although that may be covered by the min/max thing below. Unsure.

Comment (aerust) [inline soql-server-pg/src/main/scala/com/socrata/pg/query/DataSqlizerQuerier.scala]: Okay cool, was curious if there was some option hidden to have no timeout set.

**Pull request merged**